File: Entities/QuestionTypes/OpinionScaleQuestion.cs
using System.ComponentModel.DataAnnotations;
using Core.Entities;

namespace Entities.QuestionTypes
{
    public class OpinionScaleQuestion : IEntity
    {
        [Key]
        public int QuestionId { get; set; }
        public bool IsRequired { get; set; }
        public int MinScale { get; set; }
        public int MaxScale { get; set; }
        public string LabelBeginningText { get; set; }
        public string LabelMiddleText { get; set; }
        public string LabelEndText { get; set; }
    }
}

File: WebUI/UI/src/app/datas/models/question-types/opinion-scale-question.ts
export class OpinionScaleQuestion {
  constructor(questionId: number, isRequired: boolean, minScale: number, maxScale: number) {
    this.questionId = questionId;
    this.isRequired = isRequired;
    this.minScale = minScale;
    this.maxScale = maxScale;
  }

  questionId: number;
  isRequired: boolean;
  minScale: number;
  maxScale: number;
  labelBeginningText: string;
  labelMiddleText: string;
  labelEndText: string;
}
